test/typed_buffers_test.dart

```diff
@@ -185,13 +185,11 @@ void testInt(int bits, buffer(int length), {String testOn}) {
 }
 
 const List<int> intSamples = const [
-  0x10000000000000001,
-  0x10000000000000000, // 2^64
-  0x0ffffffffffffffff,
-  0xaaaaaaaaaaaaaaaa,
-  0x8000000000000001,
-  0x8000000000000000, // 2^63
```

Comment:
Consider keeping 0x8000000000000000 value if possible, as it is a corner case which is worth checking.
With the new fixed-size wrap-around integers, 0x8000000000000000 == -0x8000000000000000 == min.

Reply:
Works, added back.